bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -337,7 +337,8 @@ namespace bftEngine
 
 						send(m, currentPrimary()); // TODO(GG): add a mechanism that retransmits (otherwise we may start unnecessary view-change )
 
-						LOG_INFO_F(GL, "Sending ClientRequestMsg to current primary");
+						LOG_DEBUG_F(GL,
```

Comment:
This one is pretty rare, except in cases of a faulty primary. It may be useful to leave it at INFO for now.

Reply:
I will do so.